tests/integration/run_tests.py

```diff
@@ -1,2384 +0,0 @@
-        assert res["published"] is True, "submission is published, expected False"
-
-
-async def test_minimal_json_publication_rems(sess, project_id):
-    """Test minimal publication workflow with json submissions.
-
-    :param sess: HTTP session in which request call is made
-    :param project_id: id of the project the submission belongs to
-    """
-    submission = {
-        "name": "Minimal json publication",
-        "description": "Testing json publication with new doi endpoint",
-        "projectId": project_id,
-    }
-    submission_id = await post_submission(sess, submission)
-
-    await post_object_json(sess, "study", submission_id, "SRP000539.json")
-    ds_id = await post_object_json(sess, "dataset", submission_id, "dataset.json")
-
-    doi_data_raw = await create_request_json_data("doi", "test_doi.json")
-    await put_submission_doi(sess, submission_id, doi_data_raw)
-
-    dac_data = await create_request_json_data("dac", "dac_rems.json")
-    await put_submission_dac(sess, submission_id, dac_data)
-
-    await publish_submission(sess, submission_id)
-
-    async with sess.get(f"{submissions_url}/{submission_id}") as resp:
-        LOG.debug(f"Checking that submission {submission_id} was published")
-        res = await resp.json()
-        assert res["submissionId"] == submission_id, "expected submission id does not match"
-        assert res["published"] is True, "submission is published, expected False"
-
-    async with sess.get(f"{objects_url}/dataset/{ds_id}?submission_id={submission_id}") as resp:
-        LOG.debug(f"Checking that dataset {ds_id} in submission {submission_id} has rems data")
-        res = await resp.json()
-        assert res["accessionId"] == ds_id, "expected dataset id does not match"
-        assert "dac" in res
-        assert res["dac"]["workflowId"] == 1
-        assert res["dac"]["organizationId"] == "CSC"
-        assert "resourceId" in res["dac"]
-        assert "catalogueId" in res["dac"]
-
-
-async def test_health_check(sess):
-    """Test the health check endpoint.
-
-    :param sess: HTTP session in which request call is made
-    """
-    async with sess.get(f"{base_url}/health") as resp:
-        LOG.debug("Checking that health status is ok")
-        assert resp.status == 200, f"HTTP Status code error, got {resp.status}"
-        res = await resp.json()
-        assert res["status"] == "Ok"
-        assert res["services"]["database"]["status"] == "Ok"
-
-
-async def test_schemas_endpoint(sess):
-    """Test that schemas' endpoint return 200."""
-    for schema, expected_status in test_schemas:
-        async with sess.get(f"{schemas_url}/{schema}") as resp:
-            assert resp.status == expected_status, f"{resp.status} {schema}"
-
-
-async def main(url):
-    """Launch different test tasks and run them."""
-    mongo = Mongo(url)
-    database = mongo.db
-
-    async with aiohttp.ClientSession() as sess:
```

Comment:
i think when the migration of the test happened, this test case was missed:
we have 2 users each starts their own session and each does different operations, that is why we have
```
test_user_given = "Given"
test_user_family = "Family"
test_user = "user_given@test.what"

other_test_user_given = "Mock"
other_test_user_family = "Family"
other_test_user = "mock_user@test.what"
```
 
the idea behind this test is to check that each user is separated in the db, so they don't collide and information is not assigned to a different user, and sessions are handled properly.

Reply:
Nice catch

Reply:
Now that I took a look at it, I didn't see how it is testing those things you mentioned. Submissions are grouped by project, and not by user, so users are able to see each other's submissions anyway.

We could create an issue for testing those, but ATM, I really don't see how the change impacted the testing scope of the users.

Reply:
>  so users are able to see each other's submissions anyway.

true, however not sure they are able unless part of the same project, and we could use a test case for having separate projects and we can add that issue.

What that test did was related to sessions and it used them along with submissions to see the submissions are handled properly per session and information does go in the wrong places

Reply:
will be solved i another issue/PR